ubccr/xdmod
# Update user organization admin

## Description
Updating the code responsible for associating an XDMoD user w/ an Organization.

## Motivation and Context
We have updated the way in which we expect users to be associated to an organization. Previously a user could be associated with multiple organizations. But we found, upon inspection of our data that the only users who were using this functionality were Developers.

In an effort to simplify our code base / ways in which a user interacts with our system we have moved to requiring a user be associated with one and only one organization. This is to be accomplished by the user being associated with a 'Person'. This person's associated organization is then used to populate the Users organization.

## Tests performed
Testing Platforms: Open XDMoD && XSEDE
- Unit Tests
- Component Tests
- Integration Tests
- UI Tests
- Manual Tests ( per testing platform ): 
    - Login to the Internal Dashboard and ensure that all of the existing Users have had their organization set appropriately.
        - when testing on XSEDE the `acl-import` script must be run prior to testing.
  - Login with each existing user, ensure that the quick filters are displaying / populated as expected.

## Types of changes


## Checklist:
